.github/workflows/tests.yaml

```diff
@@ -25,19 +25,19 @@ jobs:
         export JUNIT_REPORT_DIR=$(realpath ${TARGET})
         case "${TARGET}" in
           linux-amd64-integration-1-cpu)
-            GOARCH=amd64 CPU=1 PASSES='integration' RACE='false' ./scripts/test.sh
+            GOARCH=amd64 CPU=1 PASSES='integration' RACE='false' ETCD_VERIFY=lock ./scripts/test.sh
```

Comment:
can we make ETCD_VERIFY=all the default setting set witin the `./scripts/test.sh` ?

Reply:
Didn't want to enable `all` on first CI run as I'm not familiar with CI verification. Will try to enable it after I confirm all test pass.